bodhi/server/models/models.py

```diff
@@ -2233,12 +2233,13 @@ def expire(self):
         if self.expired_date is not None:
             return
 
-        koji_session = buildsys.get_session()
-        try:
-            koji_session.untagBuild(self.build.release.override_tag,
-                                    self.build.nvr, strict=True)
-        except Exception, e:
-            log.error('Unable to untag override %s: %s' % (self.build.nvr, e))
+        if datetime.utcnow() >= self.expiration_date:
```

Comment:
I wonder if we should take a closer look at https://github.com/fedora-infra/bodhi/blob/2.3.3/bodhi/server/consumers/masher.py#L544-L609 ?